project.clj

```diff
@@ -9,6 +9,10 @@
           :defaults {:doc/format :markdown}
           :include [riddley.walk riddley.compiler]
           :output-dir "doc"}
-  :profiles {:provided {:dependencies [[org.clojure/clojure "1.8.0"]]}}
+  :profiles {:provided {:dependencies [[org.clojure/clojure "1.8.0"]]}
+             :dev {:dependencies [[org.clojure/clojure "1.10.1"]
+                                  [pjstadig/humane-test-output "0.10.0"]]
+                   :injections   [(require 'pjstadig.humane-test-output)
+                                  (pjstadig.humane-test-output/activate!)]}}
```

Comment:
Added a `:dev` profile that uses Clojure 1.10.1 by default and adds [`humane-test-output`](https://github.com/pjstadig/humane-test-output) to make the tests easier to read